### PR TITLE
[IMP]  stock_manual_transfer: Add user for default in Manual Transfers group T#79454

### PR DESCRIPTION
**Pull request by ajteposte**
>>>
By default, it is recommended that the admin and root users have all permissions due to their roles and to facilitate functional testing.
>>> 

Added the root user (`base.user_root`) and the admin user (`base.user_admin`) to the `group_stock_manual_transfer` group. Since during functional testing in the TyP module, it was detected that they were not added by default.

### CODE REVIEW

File: stock_manual_transfer/security/res_groups.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
     <record id="group_stock_manual_transfer" model="res.groups">
         <field name="name">Can make manual transfers</field>
+        <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]" />
```
Comment (luisg123v) [inline stock_manual_transfer/security/res_groups.xml]: Use the Command syntax, please

**Review by luisg123v — CHANGES_REQUESTED** [review]
Commit title should reference the module, not the repo.

File: stock_manual_transfer/security/res_groups.xml (hunk 1 above)
Comment (ajteposte) [reply stock_manual_transfer/security/res_groups.xml]: @luisg123v I make the change, is it correct how I did it?

Comment (luisg123v) [reply stock_manual_transfer/security/res_groups.xml]: Not applied correctly, I meant:

`<field name="users" eval="[Command.link(ref('base.user_root')), Command.link(ref('base.user_admin'))]" />`

For more info, read [this](https://github.com/orgs/OCA/discussions/157#discussioncomment-8753910)

**Review by luisg123v — CHANGES_REQUESTED** [review]
Regarding commit presentation:
- Module name is in plural, but the module is in singular (transfer instead of transfers)
- In author, name seems to be inverted

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Comment by luisg123v** [comment]
@TeposteAJ please submit PRs also for 15.0 and 16.0

**Pull request merged**